lib/matplotlib/mathtext.py

```diff
@@ -830,16 +858,18 @@ def _get_glyph(self, fontname, font_class, sym, fontsize, math=True):
 
         if not found_symbol:
             if self.cm_fallback:
-                if isinstance(self.cm_fallback, BakomaFonts):
-                    _log.warning(
-                        "Substituting with a symbol from Computer Modern.")
-                if (fontname in ('it', 'regular') and
-                        isinstance(self.cm_fallback, StixFonts)):
-                    return self.cm_fallback._get_glyph(
-                            'rm', font_class, sym, fontsize)
-                else:
-                    return self.cm_fallback._get_glyph(
-                        fontname, font_class, sym, fontsize)
+                if (fontname in ('it', 'regular')
+                        and isinstance(self.cm_fallback, StixFonts)):
+                    fontname = 'rm'
+
+                g = self.cm_fallback._get_glyph(fontname, font_class, sym, fontsize)
+                fname = g[0].family_name
+                if fname.startswith("cm"):
```

Comment:
perhaps a bit brittle, perhaps just special-case BakomaFonts instead?

Reply:
Seeing it didn't change in 13 years, should be fine :D 
![image](https://user-images.githubusercontent.com/13226500/74941558-857de480-53f3-11ea-8530-f6ad4be58ac0.png)

Reply:
I mean someone could later add another fallback font whose name also starts with "cm".